docs/source/Rules/Rules.rst

```diff
@@ -718,13 +718,17 @@ A really great feature to use is the internal variables. You set them like this:
 
  Let,<n>,<value>
 
+ Let,1,15
+
  Let,border,3
 
  Let,angle,0.436
 
 Where n can be a positive integer (type ``uint32_t``), or a name (identifier), and the value a floating point value. To use the values in strings you can
 either use the ``%v7%`` syntax or ``[var#7]``, or when using named variables: ``%v_border%`` or ``[var#border]``.
 
+Added: 2025-03-31: Support for named variables.
```

Comment:
Why not 1 day later? ;)

Reply:
> Why not 1 day later? ;)

Who would trust that...? 😛

Reply:
Well Google did this about 21 years ago with some email-client with 1 GByte of storage... that must have been a joke for sure!